snapshot/lib/snapshot/runner.rb

```diff
@@ -204,6 +207,21 @@ def erase_simulator(device_type)
       `xcrun simctl erase #{device_udid} &> /dev/null`
     end
 
+    def localize_simulator(device_type, language, locale)
+      apple_locale = language.sub("-", "_")
+      apple_languages = [language]
+
+      UI.message "Localizing #{device_type} (AppleLocale=#{apple_locale} AppleLanguages=[#{apple_languages}])"
+
+      device_udid = TestCommandGenerator.device_udid(device_type)
```

Comment:
This can return `nil` if the `device_type` is not found. Let's guard against going forward with the file write in that case.